src/annotator.coffee

```diff
@@ -59,9 +60,12 @@ class Annotator extends Delegator
   #     # Fallback for unsupported browsers.
   #
   # Returns a new instance of the Annotator.
-  constructor: (element, options) ->
+  constructor: (element, options={}) ->
     super
     @plugins = {}
+    @presets = @options.presets || []
+
+    this.subscribe 'annotationCreated', this.labelHighlight
```

Comment:
Lol this library has such inconsistent use of `this.` vs `@`, it was fucking with my head. I'll update all of my changes to use `@`